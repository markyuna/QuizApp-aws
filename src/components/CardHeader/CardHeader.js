import { Image, StyleSheet, Text, View } from 'react-native'
import React from 'react'
import { FontAwesome5 } from '@expo/vector-icons';

const CardHeader = () => {
  return (
    <View style={styles.container} >
        
            <FontAwesome5 name="user-circle" size={50} color="#1F9515" style={{padding: 10}}/>
            <View style={styles.user}>
                <Text>Vous</Text>
                <Text>0 pts</Text>
                <Text>Niveau 1 ( 0% )</Text>
            </View>
       
      
      <View style={styles.bag}>
        <Image 
            style={{width: 30, height: 50, padding: 10}}
            source={require('../../../assets/images/moneybag.png')}
            
        />
        <Text>30</Text>
      </View>
    </View>
  )
}

export default CardHeader

const styles = StyleSheet.create({
    container: {
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'space-between',
        borderWidth: 0,
        borderColor: 'transparent',
        borderRadius: 15,        
        backgroundColor: '#fff023',
    },
    user: {
        padding: 10,
        flexDirection: 'column',
        // borderColor: 'blue',
        // borderRightWidth: 2,
        // marginLeft: 10,
    },
    bag: {
        padding: 10,
        flexDirection: 'column',
        alignItems: 'center',
        borderRadius: 15, 
        // borderColor: 'blue',
        backgroundColor: '#fff',
    }
})