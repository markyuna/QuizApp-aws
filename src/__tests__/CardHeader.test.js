// src/components/__tests__/CardHeader.test.js
import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import CardHeader from '../components/CardHeader';

it('renders correctly', () => {
  const tree = renderer.create(<CardHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
