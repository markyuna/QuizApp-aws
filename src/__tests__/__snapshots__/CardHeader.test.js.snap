// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#fff023",
      "borderColor": "transparent",
      "borderRadius": 15,
      "borderWidth": 0,
      "flexDirection": "row",
      "justifyContent": "space-between",
    }
  }
>
  <Text />
  <View
    style={
      {
        "flexDirection": "column",
        "padding": 10,
      }
    }
  >
    <Text>
      Vous
    </Text>
    <Text>
      0 pts
    </Text>
    <Text>
      Niveau 1 ( 0% )
    </Text>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "borderRadius": 15,
        "flexDirection": "column",
        "padding": 10,
      }
    }
  >
    <Image
      source={1}
      style={
        {
          "height": 50,
          "padding": 10,
          "width": 30,
        }
      }
    />
    <Text>
      30
    </Text>
  </View>
</View>
`;
